hack/update-license.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Copyright  The Karmada Authors.
```

Comment:
Is the year field missing here?

Reply:
My mistake, these may because I use addlicense cmd without git commit, so it did not find year info from git log, I add a verification:
```shell
year=$(git log --diff-filter=A --follow --format=%aD -- "$file" | head -1 | cut -d' ' -f4)
        if [ -z "$year" ]; then
          year=$(date +%Y)
        fi
```
Thanks for pointing this!